nsls2_detector_handlers/electrometer.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import os
+import pandas as pd
+
+from . import HandlerBase
```

Comment:
Can we sort the imports (e.g., with `isort`)?

Reply:
Thanks!

Reply:
Thanks for the `isort` recommendation, hadn't encountered it before. It's good to have tools to help with being consistent about these things!

I think all the comments are addressed now, let me know if you need any more changes.